page-objects/src/components/editor/TextEditor.ts

```diff
@@ -102,6 +102,7 @@ export class TextEditor extends Editor {
     async getText(): Promise<string> {
         const inputarea = await this.findElement(TextEditor.locators.Editor.inputArea);
         await inputarea.sendKeys(Key.chord(TextEditor.ctlKey, 'a'), Key.chord(TextEditor.ctlKey, 'c'));
+        await new Promise(res => setTimeout(res, 500));
         const text = clipboard.readSync();
         await inputarea.sendKeys(Key.UP);
         clipboard.writeSync('');
```

Comment:
In case someone works on dev machine, should not we save old clipboard before using it?

eg.
```ts
const old = clipboard.readSync();

try {
     ...
}
finally {
    clipboard.writeSync(old);
}
```

Reply:
I agree, but it is out of scope for this PR. Could you please open new issue for that?

Reply:
This might be bigger than anticipated and it might require more thorough inspection of all components.

Opened #687